modules/cnf-installing-the-operators.adoc

```diff
@@ -0,0 +1,273 @@
+spec: {}
+----
+
+. Create the `operator_catalogsource.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-marketplace
+spec:
+  displayName: Openshift Performance Addon Operator
+  icon:
+    base64data: ""
+    mediatype: ""
+  image: quay.io/openshift-kni/performance-addon-operator-index:4.7-snapshot
+  publisher: Red Hat
+  sourceType: grpc
+----
+
+. Create the `operator_operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+----
+
+. Create the `operator_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: "4.7"
+  name: performance-addon-operator
+  source: performance-addon-operator
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# TODO handle namespace with var
+
+resources:
+  - operator-namespace.yaml
+  - operator_operatorgroup.yaml
+  - operator_catalogsource.yaml
+  - operator_subscription.yaml
+----
+
+[id="cnf-installing-the-precision-time-protocol-operator_{context}"]
+== Installing the Precision Time Protocol (PTP) operator
+
+Install the PTP Operator using the {product-title} CLI or the web console.
+
+.Procedure
+
+. Create the `01_namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-ptp
+  labels:
+    openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
+----
+
+. Create the `02_operator_group.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: ptp-operators
+  namespace: openshift-ptp
+spec:
+  targetNamespaces:
+    - openshift-ptp
+----
+
+. Create the `03_subscription.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ptp-operator-subscription
+  namespace: openshift-ptp
+spec:
+  channel: "4.6"
+  name: ptp-operator
+  source: "redhat-operators"
+  sourceNamespace: openshift-marketplace
+----
+
+. Configure the `kustomization.yaml` file:
++
+[source,yaml]
+----
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - 01_namespace.yaml
+  - 02_operator_group.yaml
+  - 03_subscription.yaml
+----
+
+[id="cnf-applying-the-stream-control-transmission-protocol-patch_{context}"]
+== Applying the Stream Control Transmission Protocol (SCTP) patch
+
+Load and enable the blacklisted SCTP kernel module on worker nodes in your cluster.
+
+.Procedure
+
+. Create the `is_ready.sh` executable file:
```

Comment:
Since we are describing a manual deployment here, it seems there is no need to create the is_ready.sh script. A user can just run this command manually:
oc wait mcp/worker-cnf --for condition=updated
and wait until it completes

Reply:
"Use the following command to verify the changes have been applied to the cluster (the command will block until all the modifications are done):
oc wait mcp/worker-cnf --for condition=updated"

Reply:
Changed.